src/cli.ts

```diff
@@ -32,7 +34,11 @@ export class Cli {
             // eslint-disable-next-line @typescript-eslint/no-explicit-any
         } as any;
```

Comment:
there's a `@types/umzug` package I think we should be using instead of linting suppression

Reply:
you happy with this now?